core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/GoogleProjectRegistrationComponentSpec.scala

```diff
@@ -53,4 +53,103 @@ class GoogleProjectRegistrationComponentSpec
 
   }
 
+  it should "findByIdsAndBillingProject" in withDefaultTestDatabase {
+    val billingProject = testData.testProject1
+    val googleProjectRegistrations = Seq(
+      GoogleProjectRegistration(GoogleProjectId("project1"),
+                                billingProject.billingAccount,
+                                Some("message1"),
+                                billingProject.projectName
+      ),
+      GoogleProjectRegistration(GoogleProjectId("project2"),
+                                billingProject.billingAccount,
+                                Some("message2"),
+                                billingProject.projectName
+      ),
+      GoogleProjectRegistration(GoogleProjectId("project3"),
+                                billingProject.billingAccount,
+                                Some("message3"),
+                                billingProject.projectName
+      )
+    )
+
+    googleProjectRegistrations.foreach { googleProject =>
+      runAndWait(googleProjectRegistrationQuery.create(googleProject))
+    }
+
+    val result = runAndWait(
+      googleProjectRegistrationQuery.findByIdsAndBillingProject(Set(GoogleProjectId("project1"),
+                                                                    GoogleProjectId("project2")
+                                                                ),
+                                                                None,
+                                                                10,
+                                                                0
+      )
+    )
+
+    result should have size 2
+    result.map(_.googleProjectId.value) should contain allOf ("project1", "project2")
```

Comment:
use `only` instead of `allOf` for an accurate test:
```suggestion
    result.map(_.googleProjectId.value) should contain only ("project1", "project2")
```

`Seq("foo", "bar", "baz", "qux") should contain allOf ("foo", "bar")` passes, but `Seq("foo", "bar", "baz", "qux") should contain only ("foo", "bar")` fails.

(this applies to the rest of the test too)